test/board_test.exs

```diff
@@ -1,14 +1,27 @@
 defmodule BoardTest do
   use ExUnit.Case
 
-  @empty_board Board.empty_board
+  @empty_3_by_3_board Board.empty_board
+  @empty_4_by_4_board Board.empty_board(4)
 
-  test "returns an empty board" do
-    refute Board.board_full?(@empty_board)
+  test "returns an empty 3x3 board" do
+    refute Board.board_full?(Board.empty_board)
+  end
+
+  test "3x3 board has 9 cells" do
```

Comment:
The `@empty_3_by_3_board` is calling `Board.empty`. Which means that the test wasn't self-contained. Red flag! I updated it to make it more obvious which function is called.